docs/docs/mdx/writing-pages.md

```diff
@@ -67,47 +84,123 @@ The chart is rendered inside our MDX document.
 <FAQ />
 ```
 
-## Exports
+The `<Chart />` component coming from a `.js` file would be written like any
+other React component, the `<FAQ />` component coming from an `.mdx`
+file might look something like this:
+
+<!-- prettier-ignore -->
+```markdown:title="src/content/faq.mdx"
```

Comment:
Technically this isn't a layout, it's just overriding the default layout. The pre-existing example had this `faq.mdx` file referenced from `/content` so I just left it. 

Maybe the comment makes it more confusing. The idea here is that you can include MDX documents in other MDX documents, however they're all wrapped by the default layout that's defined in the `gatsby-config`. That means if you include `faq.mdx` inside of a page like `some-guide.mdx` then you'll get this nesting doll effect with the layouts (where headers or nav bars get repeated inside of each other).

The default export makes it so that default layout isn't applied. Any suggestion on how to make it clearer?